src/main.cpp

```diff
@@ -50,6 +50,14 @@ bool sokoban_undo() {
     return soko.undo();
 }
 
+bool sokoban_redo() {
+    return soko.redo();
+}
+
+bool sokoban_goTo(unsigned int r, unsigned int c) {
```

Comment:
Shoud be snake_case, `sokoban_goto`.

`r` as `row` and `c` as `col` can't hurt here.

Since JS doesn't distinguish unisgned/signed, I'm not sure making the params unsigned will enforce much from a compiler perspective. Probably not a big deal.